azkaban-web-server/src/main/java/azkaban/webapp/AzkabanWebServer.java

```diff
@@ -296,209 +273,6 @@ private static void validateDatabaseVersion(final Props azkabanSettings)
     }
   }
 
-  private static void configureRoutes(final Server server, final Props azkabanSettings)
```

Comment:
This method moved below because of the save actions plugin. no change in logic. just parameter names were updated.